runtime/compiler/optimizer/IdiomRecognition.cpp

```diff
@@ -2302,6 +2302,9 @@ TR_CISCNode *
 TR_CISCTransformer::addAllSubNodes(TR_CISCGraph *const graph, TR::Block *const block, TR::TreeTop *const top,
                                    TR::Node *const parent, TR::Node *const node, const int32_t dagId)
    {
+   //IdiomRecognition doesn't know how to handle rdbar/wrtbar for now
+   if ((node->getOpCode().isReadBar() || node->getOpCode().isWrtBar()) && comp()->incompleteOptimizerSupportForReadWriteBarriers())
```

Comment:
can we reverse the order of the && to make this more clear and break onto a new line at the && so it looks more like
```if (comp()->incompleteOptimizerSupportForReadWriteBarriers()
       && (node->getOpCode().isReadBar() || node->getOpCode().isWrtBar()))

Reply:
changed according to the comment. Thanks!